smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/PackageManifestGenerator.kt

```diff
@@ -6,12 +6,15 @@ package software.amazon.smithy.swift.codegen
 
 import software.amazon.smithy.codegen.core.SymbolDependency
 import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
+import kotlin.jvm.optionals.getOrNull
+
+val PACKAGE_MANIFEST_NAME = "Package.swift.txt"
```

Comment:
`Package.swift` is temporarily renamed so that Xcode does not try to interpret each service client as its own Swift package just yet.